_sass/_main.scss

```diff
@@ -549,11 +549,6 @@ body {
         display: none;
       }
     }
-    @media (min-width: 600px) {
```

Comment:
Removing this will allow the `project-link` that appears directly below the description to show at all times.